feat: Introduce enableAutoDestroy() helper function (#1245)
vuejs/vue-test-utils: docs/api/enableAutoDestroy.md

## enableAutoDestroy(hook)

- **Arguments:**

  - `{Function} hook`

- **Usage:**

> `enableAutoDestroy` allows to destroy all created `Wrapper` instances using the passed hook function (for example [`afterEach`](https://jestjs.io/docs/en/api#aftereachfn-timeout)).

Reviewer: **nitpick:** Wording is a little off here.

```suggestion
`enableAutoDestroy` will destroy all created `Wrapper` instances using the passed hook function (for example [`afterEach`](https://jestjs.io/docs/en/api#aftereachfn-timeout)).
```
